### DISCUSSION TITLE
Find and replace

### DISCUSSION THREAD

**Discussion by Knah-Tsaeb** [KDE/ghostwriter#863] [Ideas]
Hi !
Can be a good idea to implement "Find and replace" functionality. GhostWriter already have "Find" function (CRTL+F) but missing "Replace" (CTRL+H).

Thanks for your work.
Have a nice day.

**Comment by wereturtle** [comment]
It does have this capability, but if you are using KDE Plasma by any chance, the shortcut gets changed to Ctrl+R. Yo be sure what the shortcut is, click on the Edit menu in the menu bar and see what is listed there. 

-- 
  Megan
  ghostwriter developer

**Comment by Knah-Tsaeb** [comment]
Oh ! I understand.

Thanks for response.
